aten/src/TH/THTensor.hpp

```diff
@@ -35,6 +36,9 @@ struct THTensor
     std::vector<int64_t> sizes_;
     std::vector<int64_t> strides_;
 
+    // TODO: get rid of this, use the sizes_/strides_ .size() instead
```

Comment:
Do you want to add a little about what else has to happen before you can fix this TODO?